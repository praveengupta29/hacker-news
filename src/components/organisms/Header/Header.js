import React from 'react';

import Anchor from '../../atoms/Anchor';
import Image from '../../atoms/Image';
import Navigation from '../../molecules/Navigation';

import './Header.scss';

const Header = () => {
  return (
    <div className="header flex">
      <Anchor to="/">
        <Image
          src="/images/logo.png"
          className="logo"
          alt="logo"
          height="18"
          width="18"
        />
      </Anchor>

      <Navigation />
    </div>
  );
};

export default Header;
