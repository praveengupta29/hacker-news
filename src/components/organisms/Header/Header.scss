@import '../../../styles/utils/colors';

.header {
  background-color: $orange;
  padding: 2px;

  .logo {
    border: 1px solid $white;
  }
}
