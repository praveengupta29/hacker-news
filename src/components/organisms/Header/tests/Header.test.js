import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header component', () => {
  it('should render correctly and match snapshot', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
