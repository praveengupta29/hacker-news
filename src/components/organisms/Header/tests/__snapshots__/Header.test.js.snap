// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly and match snapshot 1`] = `
<div
  className="header flex"
>
  <Anchor
    className=""
    handleLinkClick={[Function]}
    target="_self"
    to="/"
  >
    <Image
      alt="logo"
      className="logo"
      height="18"
      sizes=""
      src="/images/logo.png"
      srcset=""
      width="18"
    />
  </Anchor>
  <Navigation />
</div>
`;
